File: Map/src/app/shared/ToChucLongBe.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ToChucLongBe } from 'src/app/shared/ToChucLongBe.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ToChucLongBeService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ToChucLongBe";
    }
}

File: Service/Implement/ToChucLongBeService.cs
namespace Service.Implement
{
    public class ToChucLongBeService : BaseService<ToChucLongBe, IToChucLongBeRepository>
    , IToChucLongBeService
    {
    private readonly IToChucLongBeRepository _ToChucLongBeRepository;
    public ToChucLongBeService(IToChucLongBeRepository ToChucLongBeRepository) : base(ToChucLongBeRepository)
    {
    _ToChucLongBeRepository = ToChucLongBeRepository;
    }
    }
    }

